templates/custom_actions/custom_action_form.html

```diff
@@ -0,0 +1,15 @@
+{% extends "generic/object_form.html" %}
+{% block form_actions %}
+  <div class="text-sm mt-2">
+    {% if object and object.operation_ids %}
```

Comment:
I don't see `operation_ids` defined anywhere?

Reply:
Ah thanks. I can remove this whole template file